### DISCUSSION TITLE
S100105 - Not working anymore after update

### DISCUSSION THREAD

**Discussion by choumarin** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#586] [Q&A]
I had a pretty old version and everything [was working fine](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/discussions/293). But for (bad) "reasons" I decided to update the firmware. Now the board doesn't seem to even boot (doesn't join wifi / doesn't show the AP and display stays off). 
I've plugged it to computer and it boots and everything looks ok, I've configured both pump and display as 2021.
I added the 10k resistor because that seems to be "recommended" with the blue LLC but no difference.

Any pointers?

**Comment by visualapproach** [comment]
Not really... did you set pcb type in hwconfig and saved? Go through all ui pages and resave.

**Comment by choumarin** [comment]
So for some reason it seems that the GFCI plug failed, and maybe that's connected? I replaced it and everything is working now. strange but 🤷 
working great now
